src/everest_models/jobs/fm_well_trajectory/well_trajectory_resinsight.py

```diff
@@ -42,7 +48,7 @@ def __exit__(self, *_) -> None:
 
 
 def _save_project(project_path: str, project: rips.Project):
-    project_file = str(project_path / "model.rsp")
+    project_file = str(project_path / ".model.rsp")
```

Comment:
This path looks a bit strange to me. Why is the extra `.` needed ?

Reply:
The previous version of the forward model was reviewed by Equinor to be qualified as TRL4, and they asked that this file would be hidden to the user. Hence the `.`.

Reply:
When qualifying the model, we had different expectations for the generated resinsight project. To avoid misuse or misinterpretation of the job, we decided to hide it.